paddle/fluid/pybind/pybind.cc

```diff
@@ -949,8 +951,10 @@ All parameter, weight, gradient are variables in Paddle.
                      int block_id, bool create_local_scope, bool create_vars,
                      const std::vector<std::string> &fetch_vars) {
         pybind11::gil_scoped_release release;
+        paddle::platform::SetNumThreads(FLAGS_paddle_num_threads);
```

Comment:
Using the flag `FLAGS_paddle_num_threads` in every call to the `run` method seems inappropriate to me.
What about adding a pybind to the `SetNumThreads()` method, and calling it in the test before executor's `run`?
@luotao1 , what do you think about such an idea?

Reply:
> What about adding a pybind to the SetNumThreads() method, and calling it in the test before executor's run

I think it is a nice idea.

Reply:
OK I've changed that